pkg/envtest/crd.go

```diff
@@ -27,6 +27,7 @@ import (
 
 	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I guess this line was making `goimports` unhappy.
To be consistent with other k8s projects, we should alias it as `metav1`. I believe you already know it. :)